public/pages/Overview/components/Widgets/DetectorsWidget.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
nit: Only need one of these in each file

Reply:
done